app/models/appointment.rb

```diff
@@ -40,6 +40,7 @@ class Appointment < ApplicationRecord
   validate :cancel_reason_is_present_if_cancelled
   validates :device_created_at, presence: true
   validates :device_updated_at, presence: true
+  validates :appointment_type, presence: true
```

Comment:
I added this because I got database errors instead of validation errors when I tried saving an appointment.